### DISCUSSION TITLE
Performance monitoring tool for Wrappid Framework

### DISCUSSION THREAD

**Discussion by anantakumarghosh** [wrappid/.github#5] [Ideas]
Proposing implementation of Sentry([Sentry.io](https://sentry.io)), an opensource tool. to monitor performance of our framework. 

Take a look @wrappid/core-team

**Comment by saifalamkhan373** [comment]
It's awesome!!
**Here is the use of Sentry Software: It is an open-source, cross-platform error tracking tool that helps developers to monitor, and fix crashes in real-time. Its major features help in boosting efficiency and improve the user experience like tracking and detecting every possible exception from its SDK; whether it is handled by you or the code block.**

**Comment by techoneel** [comment]
The following links will be helpful
https://github.com/getsentry/sentry-javascript
https://github.com/getsentry/sentry-react-native

**Reply by anantakumarghosh** [reply]
> The following links will be helpful https://github.com/getsentry/sentry-javascript https://github.com/getsentry/sentry-react-native

Thanks
